src/reducers/greetingReducer.js

```diff
@@ -0,0 +1,26 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+
+const GreetingUrl = 'http://localhost:3000';
+const Method = '/get/messages';
+const initialState = {};
+
+export const GreetingsThunk = createAsyncThunk(Method, async () => {
+  const greeting = await (await fetch(`${GreetingUrl}`)).json();
+  return greeting;
+});
+
+const GreetingSlice = createSlice({
+  name: 'Greetings',
+  initialState,
+  reducers: {},
+  extraReducers: {
+    [GreetingsThunk.fulfilled]: (state, action) => {
+      const { payload } = action;
+      return {
+        ...payload,
+      };
+    },
+  },
+});
```

Comment:
- [ ] **[OPTIONAL]** Please consider using the new suggested syntax for the `extraReducers` which is using the `builder` param. I'm marking this as OPTIONAL but I strongly suggest you apply this change as this syntax is soon getting deprecated! 🤓 🔧